src/common/ip-address.schema.json

```
@@ -0,0 +1,5 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "type": "string",
+    "pattern": "^(?:(?:^|\\.)(?:2(?:5[0-5]|[0-4]\\d)|1?\\d?\\d)){4}$"
```

Try this:

```suggestion
    "pattern": "^(?:(?:2(?:5[0-5]|[0-4]\\d)|1?\\d?\\d)(?:^|\\.)?){3}(?:2(?:5[0-5]|[0-4]\\d)|1?\\d?\\d)$"
```

https://regex101.com/r/kf8qoc/1

The current pattern allows for `.1.2.3.4`